server/http_handler.go

```diff
@@ -743,6 +743,17 @@ func (h settingsHandler) ServeHTTP(w http.ResponseWriter, req *http.Request) {
 				return
 			}
 		}
+		if treadOldVersionUTF8AsUTF8MB4 := req.Form.Get("treat-old-version-utf8-as-utf8mb4"); treadOldVersionUTF8AsUTF8MB4 != "" {
```

Comment:
add a session variable to set this config.

Reply:
done.

Reply:
But the "session variable" actually is system scope, not session.

Reply:
delete the session variable for need reload schema after change this variable.